UI/data/themes/Acri.qss

```diff
@@ -533,39 +537,38 @@ QSlider::handle:vertical {
 		stop: 0.25 rgb(200,199,200),
 		stop: 1 rgb(162,161,162));
 	border: 1px solid rgb(24,24,25);
-	border-radius: 4px;
-	width: 10px;
-	height: 18px;
-	margin: -3px 0; /* handle is placed by default on the contents rect of the groove. Expand outside the groove */
+    border-radius: 3px;
+    width: 10px;
+    height: 18px;
+    margin: 0 -3px; /* handle is placed by default on the contents rect of the groove. Expand outside the groove */
```

Comment:
Just a tiny indent inconsistency here, you'll notice the indent of these four lines is inconsistent with the previous lines.  Make sure to double-check your commit diffs in the future.